src/CarsDataTable.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
+import Car from "./Car";
+import Titles from "./Titles";
+import "./CarsDataTable.css";
 
 const CarsDataTable = (props) => {
+    const allCars = props.cars.map((car, index) => (
+        <tr key={index}><Car carParams={car} /></tr>
+    ))
+const tableStyle = {border: "2px solid grey", textAlign: "center", padding: 0}
+const divStyle = {textAlign: "center"}
   return (
-    <div>
+    <div style={divStyle} ClassName="table">
       The list has {props.cars.length} cars.<br />
-      <b>Your task is to render a table here</b>.
+      <b>Your task is to render a table here </b>.
+        <table style={tableStyle}>
+            <Titles />
+            {allCars}
```

Comment:
`<tr>` cannot be a direct child of `<table>`. If you open a browser console, you'll see warnings about that. Add a `<tbody>` to your code to match the DOM tree generated by the browser.